examples/ping-ring/main.go

```diff
@@ -95,12 +95,19 @@ func main() {
 
 	logger := log.StandardLogger()
 
+	rp, err := ringpop.New("ping-app",
+		ringpop.Channel(ch),
```

Comment:
I did initially consider putting options into their own package, which would result in an API like:

```
ringpop.New(ringpop.options.Identity("..."))
```

However in the end I opted for brevity and simplicity of layout. If you think `ringpop.Identity` and `ringpop.Channel` is confusing or unclear though, I think that's a strong case to move the options modifiers into their own package.